widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -291,3 +303,25 @@ export const MoreStep = styled('div', {
     },
   },
 });
+
+export const Tag = styled('div', {
+  padding: '$5 $10',
+  borderRadius: '$md',
+  fontSize: '$12',
+  fontWeight: '$regular',
+  variants: {
+    recommended: {
+      true: {
+        backgroundColor: '$secondary',
+        color: '$background',
+        [`.${darkTheme} &`]: {
+          color: '$foreground',
+        },
+      },
+      false: {
+        border: '1px solid $secondary',
+        color: '$secondary',
+      },
+    },
+  },
+});
```

Comment:
To keep the sizes consistent, it'd be better to add borders to both versions and just switch up the background color.

Reply:
fixed